tests/test_other.py

```diff
@@ -9045,9 +9044,7 @@ def fail(args, details):
 
     # plain -O0
     legalization_message = 'to disable int64 legalization (which requires changes after link) use -s WASM_BIGINT'
-    longjmp_message = 'to disable longjmp support (which requires changes after link) use -s SUPPORT_LONGJMP=0'
     fail([], legalization_message)
-    fail(['-sWASM_BIGINT'], longjmp_message)
     fail(['-sSUPPORT_LONGJMP=0'], legalization_message)
```

Comment:
i think this line is not needed any more

Reply:
Removed.